src/kestrel/utils.py

```diff
@@ -60,6 +60,21 @@ def mkdtemp():
     return p
 
 
+def replace_path_substring(path, oldstring, newstring):
+    p = pathlib.Path(str(path).replace(oldstring, newstring))
+    return p
+
+
+def make_ingest_stixbundle_filepath(ingestfile):
```

Comment:
Could we move this function from `/src/kestrel/*` to `/src/kestrel_datasource_stixshifter/*`? These are two Python packages, which should have minimal dependencies. You may just put it in `interface.py` if the function is only used once. Or a `utils.py` module if the function is used by multiple modules in `kestrel_datasource_stixshifter`.